packages/Webkul/Category/src/Repositories/CategoryRepository.php

```diff
@@ -73,6 +74,7 @@ public function update(array $data, $id, $attribute = 'id')
         $category->update($data);
 
         $this->uploadImages($data, $category);
+        $this->uploadBanner($data,$category);
```

Comment:
@Rachana1005 

Why, we are using new method, we can use the $type param for the banner image.